### ISSUE TITLE
Blurry text on 1080p monitor

### ISSUE THREAD

**Issue by zangaby** [jimradford/superputty#634]
Hello,

Is there any GUI settings I need to check to avoid having blurry text on the interface and on the sessions ?
If I just open Putty normally it's all fine, but in superputty everything is blurry. 

Thank you for your help !

**Comment by Maxiride** [comment]
I just recently needed tab managment in putty and found this application useful, however, I am too having intense blur in the application plus it looks like everything is zoomed in.

I am using a 1920x1080 monitor. For the sake of "debugging" Putty looks fine instead.

**Comment by marbaa** [comment]
Blurines could be caused by DPI in Windows. Are you both using different value than 100%?

**Comment by Maxiride** [comment]
> Blurines could be caused by DPI in Windows. Are you both using different value than 100%?

Yes I am using 125% (one of the preset values in the dropdown menu), furthermore in the related advanced settings the option "Let windows try to fix apps so they're not blurry" is enabled.

I can confirm that setting the scaling to 100% makes the text appear normal size (as same in Putty) and not blurry.
Setting the scaling to 125% and disabling the "Let windows try to fix apps so they're not blurry" option leaves the app blurry.

I've also checked with putty the settings and it seems to not be affected by them, the text is always sharp (and if zoomed as it should be due to the scale setting).

**Comment by RyanQuey** [comment]
Setting DPI to 100% fixes super putty for me as well, but then the rest of Windows looks off. It would be nice if there were a way to fix this

**Comment by bjansen** [comment]
Setting the scaling behavior to 'Application' in the .exe properties fixes the bluriness:

![image](https://user-images.githubusercontent.com/281528/98684100-1328a800-2366-11eb-9746-6f82c1d0e789.png)

But then the settings dialogs look 'shrinked':

![image](https://user-images.githubusercontent.com/281528/98683940-e5dbfa00-2365-11eb-91ec-7a9d731b7368.png)

Still better than before, imo.

**Comment by bjansen** [comment]
I submitted a PR to try and fix the problem: #816
